CHANGELOG.md

```diff
@@ -2,12 +2,24 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### Unreleased
```

Comment:
I think you can remove this. As long as you follow conventional commit messages then it should appear in the changelog on release https://www.conventionalcommits.org/en/v1.0.0/